packages/git/src/browser/history/git-commit-detail-open-handler.ts

```diff
@@ -46,10 +46,6 @@ export class GitCommitDetailOpenHandler extends WidgetOpenHandler<GitCommitDetai
         }
     }
 
-    protected async doOpen(widget: GitCommitDetailWidget, options: GitCommitDetailOpenerOptions): Promise<void> {
-        await super.doOpen(widget, options);
-    }
-
```

Comment:
Why is this change necessary?

Reply:
Since number of parameters of `super.doOpen()` is changed, this override method also needs to be changed.
However, I believe this override method has nothing different in behavior from method `super.doOpen()` because it just calling `super.doOpen()`.
So I just removed it.